client/angular.json

```diff
@@ -40,13 +35,13 @@
               "budgets": [
                 {
                   "type": "initial",
-                  "maximumWarning": "500kb",
-                  "maximumError": "1mb"
+                  "maximumWarning": "5mb",
```

Comment:
Haha this is probably good, I kept hitting the 500kb error on the tutorial app!

Reply:
(btw nit: if you're making a change like this that you didn't explain in the commit message and that's mixed in with a lot of other changes due to installing `ng-bootstrap`, it's easy for reviewers to miss. I would recommend either mentioning it in the commit message or adding an inline comment on the code in GitHub to mention what you're doing here.)